cmd/etcd-migrate/main.go

```
@@ -66,6 +70,16 @@ explicit --skip-backup.`,
 		},
 	}
 	bindFlags(rootCmd, cfg)
+	// A `version` subcommand rather than a --version flag: --version is already
+	// taken by the etcd-version override (see bindFlags).
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "version",
+		Short: "Print the etcd-migrate binary version",
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, _ []string) {
+			fmt.Fprintln(cmd.OutOrStdout(), version)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)